.gitignore

```diff
@@ -25,6 +25,7 @@ out
 .component-props
 
 ## env
+docs/.env
```

Comment:
Why is this necessary. the local env should be enough, no?

Reply:
no didn't work for me